com.unity.ml-agents/Runtime/Actuators/ActionSpec.cs

```diff
@@ -59,16 +65,34 @@ public static ActionSpec MakeContinuous(int numActions)
         public static ActionSpec MakeDiscrete(params int[] branchSizes)
         {
             var numActions = branchSizes.Length;
-            var actuatorSpace = new ActionSpec(0, numActions, branchSizes);
+            var actuatorSpace = new ActionSpec(0, branchSizes);
             return actuatorSpace;
         }
 
-        internal ActionSpec(int numContinuousActions, int numDiscreteActions, int[] branchSizes = null)
+        /// <summary>
+        /// Set action size fields related to continuous actions.
+        /// </summary>
+        /// <param name="numContinuousActions">Number of Continuous Actions</param>
+        public void SetContinuous(int numContinuousActions)
```

Comment:
internal?

If you keep this public, I think it should m_BranchSizes to null. As long it's only used in the one or two places that it currently is, it's probably OK as is.

Reply:
I might have not made it clear: the idea of this method is to modify the continuous part of actionSpec while keep the discrete part unchanged, so that continuous and discrete size can be configured respectively in editor.

Made it internal.